prediction_market_agent/agents/microchain_agent/nft_treasury_game/deploy_nft_treasury_game.py

```diff
@@ -34,7 +38,22 @@ class DeployableAgentNFTGameAbstract(DeployableMicrochainAgentAbstract):
 
     name: str
     wallet_address: ChecksumAddress
-    mech_address: ChecksumAddress
+    mech_address: ChecksumAddress | None
```

Comment:
```suggestion
    mech_address: ChecksumAddress | None = None
```

Otherwise it will error out because it won't have a value